go/test/apis/git_recognizer_test.go

```diff
@@ -17,61 +17,53 @@ import (
 	"github.com/redhat-developer/alizer/go/test"
 )
 
+const (
+	repoUrl    = "https://github.com/thepetk/alizer-test-resources.git"
```

Comment:
So this will need to be updated after the test resource pr goes in first right?
https://github.com/devfile-resources/alizer-test-resources/pull/1

Reply:
the `repoUrl`? I don't think so. Both commit hash and repo URL will remain the same before and after the merge. Maybe I'm missing something here?

Reply:
The current repoUrl points to your fork, would we need to change it (and also the hash) to:
```suggestion
	repoUrl    = "https://github.com/devfile-resources/alizer-test-resources.git"
```

Reply:
:D Aparently I was missing something